### ISSUE TITLE
Does not work with ipbynb files (jupyter Notebooks)?

### ISSUE THREAD

**Issue by AP3dr** [mkxml/vscode-filesize#49]
Doesn't seem to work with ipbynb files (jupyter Notebooks), right?

**Comment by mkxml** [comment]
Yes @AP3dr, you're correct, ipynb files are currently not supported. Jupyter notebooks are special files that open on a special interactive editor (not a regular vscode editor).

I will look into supporting this soon.

**Comment by toiletbowlkuku** [comment]
Perhaps you would like to get some inspiration from this extension that does work with jupyter notebooks: https://github.com/zhcode-fun/file-size
